packages/x-bus/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "x-bus",
+  "version": "1.0.0-alpha.00",
+  "description": "Event bus to help with events orchestration",
+  "author": "Empathy Systems Corporation S.L.",
+  "homepage": "https://github.com/empathyco/x/tree/main/packages/x-bus#readme",
+  "license": "Apache-2.0",
+  "main": "dist/cjs/index.js",
+  "keywords": [
+    "events",
+    "orchestration"
+  ],
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/empathyco/x.git",
+    "directory": "packages/x-bus"
+  },
+  "engines": {
+    "node": "16"
+  },
+  "scripts": {
+    "prebuild": "rimraf dist ./*.tgz",
+    "build": "concurrently \"npm run build:*\"",
+    "build:cjs": "tsc --project tsconfig.cjs.json",
+    "build:esm": "tsc --project tsconfig.esm.json",
+    "postbuild": "npm pack",
+    "test": "jest",
+    "prepublishOnly": "npm run build"
+  },
+  "bugs": {
+    "url": "https://github.com/empathyco/x/issues"
+  },
+  "devDependencies": {
```

Comment:
Good catch! I think currently is working because `rimraf` is listed as dep in other packages but we shouldn't rely on that. I've added it to the devDeps